common/storage/sql/statements.c

```diff
@@ -127,3 +127,27 @@ retcode_t iota_transactions_update_statement(const char *index_col,
                                              const iota_transaction_t tx,
                                              char statement[],
                                              size_t statement_cap) {}
+
+retcode_t iota_transactions_select_hashes_statement(const char *index_col,
+                                                    const trit_array_p key,
```

Comment:
Let's use an `enum` for this kind of key?

Reply:
But the encoding can be different, so the enumToKeyTable (if that is what you have in mind) will need entry per column per encoding (1/3/4/5)

Reply:
Nevermind, I was confusing index_col & key